src/test/java/api/support/builders/ItemBuilder.java

```diff
@@ -34,14 +35,14 @@
   private final String itemLevelCallNumberPrefix;
   private final String itemLevelCallNumberSuffix;
   private String enumeration;
-  private List<String> copyNumbers;
+  private String copyNumber;
   private List<String> yearCaption;
   private String volume;
   private final String chronology;
 
   public ItemBuilder() {
     this(UUID.randomUUID(), null, "565578437802", AVAILABLE,
-      null, null, null, null, null, null, null, null, null, null, null, null,
+      null, null, null, null, null, null, null, null, null, null, null, Collections.emptyList(),
```

Comment:
What parameter is an empty collection being passed to?

Reply:
It is `yearCaption` which is an array.

Reply:
Ok, thanks